### PR TITLE
add _.endsWith and add tests for both endsWith and startsWith

### PR DESCRIPTION
**Pull request by tombiju**
- Added in `_.endsWith` to `README.md` and added tests for both `endsWith` and `startsWith`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -171,6 +171,7 @@ then Lodash/Underscore is the better option.*
 1. [_.replace](#_replace)
 1. [_.split](#_split)
 1. [_.startsWith](#startsWith)
+1. [_.endsWith](#endsWith)
```
Comment (cht8687) [inline README.md]: could you resolve the alphatic order?  ( This item can be placed on top of _.padStart )

**Comment by cht8687** [comment]
@tombiju thanks for the PR! 👍 
could you resolve the conflicts so that we can merge. 😄

File: README.md (hunk 1 above)
Comment (tombiju) [reply README.md]: just changed the order!

**Comment by tombiju** [comment]
@cht8687 resolved the conflicts and changed the order to be alphabetic!

File: README.md [hunk 2]
```diff
@@ -166,16 +166,17 @@ then Lodash/Underscore is the better option.*
...
 1. [_.padStart and _.padEnd](#_padstart-and-_padend)
 1. [_.repeat](#_repeat)
 1. [_.replace](#_replace)
 1. [_.split](#_split)
 1. [_.startsWith](#startsWith)
-1. [_.template](#_template)
-1. [_.toLower](#_tolower)
-1. [_.toUpper](#_toupper)
+2. [_.template](#_template)
```
Comment (stevemao) [inline README.md]: Why changing these numbers?

Comment (tombiju) [reply README.md]: @stevemao fixed it, must have auto incremented from my IDE when I was moving endsWith to be alphabetically in order

**Pull request merged**